src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4592,8 +4592,8 @@ static void Test(string[] args)
         }
 
         [WorkItem(976, "https://github.com/dotnet/roslyn/issues/976")]
-        [Fact(Skip = "PROTOTYPE(CONSTISTR) - Expected changed var into a private const."), Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
-        public async Task TestNoConstantForInterpolatedStrings2()
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public async Task TestConstantForInterpolatedStrings()
```

Comment:
could you add a test where the interpolated string references a real constant value?  i.e. `0` instead of just the unbound `s`?  thanks!

Reply:
Note: this would have to be `([|$""Text{""0""}""|])`, as all constant values inside must be constant string expressions or it's not a constant string itself.

Reply:
Yup, added a test for that. A int constant should register as invalid and cause us to suggest a variable instead.

Reply:
I think Cyrus wanted a test for when the interpolation includes things that would still make it a constant value.